idunn/blocks/services_and_information.py

```diff
@@ -94,11 +94,64 @@ def from_es(cls, es_poi, lang):
         return cls(beers=beers)
 
 
+class Diet(types.Type):
+    STATUS_YES = "yes"
+    STATUS_NO = "no"
+    STATUS_ONLY = "only"
+    STATUS_UNKNOWN = "unknown"
+
+    name = validators.String()
+    value = validators.String(
+        enum=[STATUS_YES, STATUS_NO, STATUS_ONLY, STATUS_UNKNOWN]
+    )
+
+    @classmethod
+    def from_es(cls, diet_kind, data):
+        info = data.get("properties", {}).get("diet:{}".format(diet_kind))
+        value = {
+            cls.STATUS_YES: cls.STATUS_YES,
+            cls.STATUS_NO: cls.STATUS_NO,
+            cls.STATUS_ONLY: cls.STATUS_ONLY
+        }.get(info, cls.STATUS_UNKNOWN)
+        return cls(name=diet_kind, value=value)
+
+
+class Cuisine(types.Type):
+    name = validators.String()
+
+
+class CuisineBlock(BaseBlock):
+    BLOCK_TYPE = "cuisine"
+    SUPPORTED_DIETS = ("vegetarian", "vegan", "gluten_free")
+
+    cuisines = validators.Array(items=Cuisine)
+    vegetarian_status = validators.Object()
+    vegan_status = validators.Object()
+    gluten_free_status = validators.Object()
```

Comment:
Any reason to use an object here ? 
Wouldn't `"vegetarian": STATUS_YES` be more simple, with a string value among all statuses ?